extension/chrome/settings/modules/contacts.ts

```diff
@@ -118,11 +118,7 @@ View.run(class ContactsView extends View {
     const [contact] = await ContactStore.get(undefined, [$(viewPubkeyButton).closest('tr').attr('email')!]); // defined above
     $('.hide_when_rendering_subpage').css('display', 'none');
     Xss.sanitizeRender('h1', `${this.backBtn}${this.space}${contact!.email}`); // should exist - from list of contacts
-    if (contact!.client === 'cryptup') {
-      Xss.sanitizeAppend('h1', '&nbsp;&nbsp;&nbsp;&nbsp;<img src="/img/logo/flowcrypt-logo-19-19.png" />');
-    } else {
-      Xss.sanitizeAppend('h1', '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;');
-    }
+    Xss.sanitizeAppend('h1', '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'); // todo: do we need this line?
```

Comment:
I think it's for some formatting. @limonte would probably want to murder me for this one :D but for now please add it into the `Xss.sanitizeRender('h1'` on line 120.